index.js

```diff
@@ -67,10 +68,20 @@ const main = async () => {
     testSuiteCollections, 
     testCases
   }
+
   core.setOutput('repository', result);
 
-  await fs.writeFile('repository.json', JSON.stringify(result));
-  //TODO: get signed URL from TestOps and upload result to S3
+  const jsonFile = await fs.writeFile('repository.json', JSON.stringify(result))
+  .then(() => fs.readFile('repository.json', 'utf-8'));
+
+  const GITHUB_URL = core.getInput('github-url');
```

Comment:
what is github-url? `https://github.com` ?
all caps should be for constant, a local variable should be named in camel case